src/v2/view-builder/views/ov/ChallengeAuthenticatorDataOktaVerifyView.js

```diff
@@ -4,13 +4,18 @@ import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
 import AuthenticatorVerifyOptions from '../../components/AuthenticatorVerifyOptions';
 import { getAuthenticatorDataForVerification } from '../../utils/AuthenticatorUtil';
 import { Body as SelectAuthenticatorVerifyViewBody } from '../SelectAuthenticatorVerifyView';
-import { AUTHENTICATOR_KEY } from '../../../ion/RemediationConstants';
+import { AUTHENTICATOR_KEY, FORMS } from '../../../ion/RemediationConstants';
 
 const Body = SelectAuthenticatorVerifyViewBody.extend({
   getUISchema() {
     // Change the UI schema to not display radios here.
     const uiSchemas = BaseForm.prototype.getUISchema.apply(this, arguments);
     const methodsSchema = uiSchemas.find(schema => schema.name === 'authenticator.methodType');
+
+    if (this.options.currentViewState && this.options.currentViewState.name === FORMS.AUTHENTICATOR_VERIFICATION_DATA) {
```

Comment:
Hi Gowthami, I thought about it during the implementation. In my opinion, we should change the order of `methodsSchema.options` permanently rather than sorting the result of the `map` method derived from `methodsSchema.options`.

I think we can remove the name check. It's quite redundant.